ildcp-core/src/test/java/org/interledger/ildcp/IldcpResponsePacketHandlerTest.java

```diff
@@ -101,6 +101,7 @@ protected void handleIldcpResponsePacket(IldcpResponsePacket ildcpResponsePacket
 
       @Override
       protected void handleIldcpErrorPacket(InterledgerRejectPacket ildcpErrorPacket) {
+        fail("Cannot handle fulfill packets");
```

Comment:
Are the exceptions still necessary on these?

Reply:
Codacy was flagging these tests to have an assert or fail. This was just to silence it, shouldn't really hurt the correctness 😄

Reply:
Ah yes, I'm familiar with this type of requisite madness. Thanks Codacy!